src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -103,6 +103,11 @@ class PalantirDockerPlugin implements Plugin<Project> {
                     buildCommandLine.addAll('--build-arg', "${buildArg.getKey()}=${buildArg.getValue()}")
                 }
             }
+            if (!ext.labels.isEmpty()) {
+                for (Map.Entry<String, String> label : ext.labels.entrySet()) {
+                    buildCommandLine.addAll('--label', "${label.getKey()}=${label.getValue()}")
```

Comment:
Quoting the value leads to those quotes getting escaped and wrapped in with their own quotes in the labels that get stored:

```
$ docker inspect --format '{{json .Config.Labels}}' id10-quoted
{"another.label":"\"another.value\"","test-label":"\"test-value\""}

$ docker inspect --format '{{json .Config.Labels}}' id10-unquoted
{"another.label":"another.value","test-label":"test-value"}

$ docker inspect --format '{{json .Config.Labels}}' test-manifest-quoted
{"manifest": "\"{\"manifest-version\":\"1.0\",\"product-type\":\"service.v1\",...}\""}
```